Source/Hurl.RulesetManager/Windows/EditRuleset.xaml

```diff
@@ -0,0 +1,183 @@
+﻿<ui:FluentWindow
+    x:Class="Hurl.RulesetManager.Windows.EditRuleset"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="clr-namespace:Hurl.RulesetManager.Windows"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:ui="http://schemas.lepo.co/wpfui/2022/xaml"
+    Title="EditRuleset"
+    Width="800"
+    Height="450"
+    ExtendsContentIntoTitleBar="True"
+    WindowBackdropType="Mica"
+    WindowCornerPreference="Round"
+    WindowStartupLocation="CenterScreen"
+    mc:Ignorable="d">
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition />
+            <ColumnDefinition
+                Width="4*"
+                MinWidth="500"
+                MaxWidth="800" />
+            <ColumnDefinition />
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition />
+        </Grid.RowDefinitions>
+
+        <ui:TitleBar
+            Title="Ruleset Manager - Hurl"
+            Grid.ColumnSpan="3"
+            ShowMinimize="False" />
+
+        <ScrollViewer
+            Grid.Row="1"
+            Grid.Column="1"
+            Padding="12,8">
```

Comment:
In my experience, a scrollviewer with padding gives less-than-ideal results. Especially for the top and bottom. Setting margins on the content might be a bit better.